Makefile

```diff
@@ -143,7 +143,7 @@ help:
 	@echo "  userguide         to build HTML version of the users guide"
 	@echo "  python_packages   to build natcap.invest wheel and source distributions"
 	@echo "  windows_installer to build an NSIS installer for distribution"
-	@echo "  mac_installer     to build a disk image for distribution"
+	@echo "  mac_dmg     to build a disk image for distribution"
```

Comment:
Sorry about my OCD here, but could that line be indented to match the others?

Reply:
Good catch!